bundle/config/validate/fast_validate.go

```diff
@@ -30,6 +30,7 @@ func (f *fastValidateReadonly) Apply(ctx context.Context, rb bundle.ReadOnlyBund
 		JobClusterKeyDefined(),
 		JobTaskClusterSpec(),
 		SingleNodeCluster(),
+		IncludesOutsideRoot(),
```

Comment:
Does this mean that `ProcessInclude` correctly processed such files and merged into main config? So after the merge there's still an include section which is visible in validate JSON output? If so, can we catch it earlier and error out in `ProcessInclude` itself?

Reply:
Thanks, that's a good point. The downside is that with a separate mutator, you just get one warning while having it in `ProcessInclude` would lead to multiple.

However, having this located in `ProcessInclude` is cleaner and easier to maintain, so I changed the approach.